src/README.md

```diff
@@ -1,104 +1,168 @@
 [572/572] Linking CXX executable bin/ot-ncp-ftd
 + cd <path-to-ot-efr32>
 ```
 
-After a successful build, the `elf` files are found in `<path-to-ot-efr32>/build/efr32mg12/bin`.
+After a successful build, the `elf` files are found in `<path-to-ot-efr32>/build/<platform>/bin`.
 
 ```bash
-$ ls build/efr32mg12/bin
+$ ls build/$platform/bin
 ot-cli-ftd      ot-cli-mtd      ot-ncp-ftd      ot-ncp-mtd      ot-rcp      sleepy-demo-ftd      sleepy-demo-mtd
 ot-cli-ftd.s37  ot-cli-mtd.s37  ot-ncp-ftd.s37  ot-ncp-mtd.s37  ot-rcp.s37  sleepy-demo-ftd.s37  sleepy-demo-mtd.s37
 ```
 
-## Flash Binaries
+<a name="flash"/>
 
-Compiled binaries may be flashed onto the EFR32 using [JLinkGDBServer][jlinkgdbserver]. EFR32 Starter kit mainboard integrates an on-board SEGGER J-Link debugger.
+## Flashing Binaries
 
-[jlinkgdbserver]: https://www.segger.com/jlink-gdb-server.html
+Compiled binaries may be flashed onto the EFR32 using various tools from the [J-Link][j-link] software pack. EFR32 Starter kit mainboard integrates an on-board SEGGER J-Link debugger.
+
+<a name="simplicity-commander"/>
+
+### Simplicity Commander
+
+Simplicity Commander provides a graphical interface for J-Link Commander. It's included as part of [Simplicity Studio][simplicity-studio] and is also available as a [standalone application][simplicity-commander]
+
+[simplicity-commander]: https://www.silabs.com/mcu/programming-options#programming
 
 ```bash
-$ cd <path-to-JLinkGDBServer>
-$ sudo ./JLinkGDBServer -if swd -device EFR32MG12PxxxF1024
-$ cd <path-to-ot-efr32>/build/bin
-$ arm-none-eabi-gdb ot-cli-ftd
-$ (gdb) target remote 127.0.0.1:2331
-$ (gdb) load
-$ (gdb) monitor reset
-$ (gdb) c
+$ <path-to-simplicity-studio>/developer/adapter_packs/commander/commander
 ```
 
-Note: Support for the "EFR32MG12PxxxF1024" device was added to JLinkGDBServer V6.14d.
+In the J-Link Device drop-down list select the serial number of the device to flash. Click the Adapter Connect button. Ensure the Debug Interface drop-down list is set to SWD and click the Target Connect button. Click on the Flash icon on the left side of the window to switch to the flash page. In the Flash MCU pane enter the path of the ot-cli-ftd.s37 file or choose the file with the Browse... button. Click the Flash button located under the Browse... button.
+
+For more information see [UG162: Simplicity Commander Reference][ug162]
+
+[ug162]: https://www.silabs.com/documents/public/user-guides/ug162-simplicity-commander-reference-guide.pdf
+
+<a name="jlink-commander"/>
 
-Or Compiled binaries also may be flashed onto the specified EFR32 dev board using [J-Link Commander][j-link-commander].
+### J-Link Commander
+
+Compiled binaries also may be flashed onto the specified EFR32 dev board using [J-Link Commander][j-link-commander].
 
 [j-link-commander]: https://www.segger.com/products/debug-probes/j-link/tools/j-link-commander/
 
+Refer to the [table](#jlink-efr32-devices) above for which value to use for `<efr32-device>`
```

Comment:
Is `jlink-efr32-devices` the proper link? It seems to be anchored below, rather than above.